Source/AliveLibAO/Elum.cpp

```diff
@@ -313,6 +313,20 @@ __int16 Elum::VTakeDamage_411020(BaseGameObject* pFrom)
     return 1;
 }
 
+void Elum::toKnockback()
+{
+    field_B4_velx = FP_FromInteger(0);
+    field_FC_current_motion = eElumStates::State_50_Knockback_415DC0;
+
+    if (field_F4_pLine)
+    {
+        MoveOnLine_412580(0);
+    }
+    field_A8_xpos -= field_B4_velx;
```

Comment:
This seems a bit odd - I guess that MoveOnLine_412580 changes field_B4_velx otherwise it always subtracts zero which does nothing?

Reply:
you're right it subtracts zero which does nothing